dashboard/src/routes/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { Route, Routes as RouterRoutes } from 'react-router-dom';
```

Comment:
Could be using `@tanstack/router`. Seems very cool and now that you're starting over somewhere

Reply:
hmmm quite an interesting idea but not this time